hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-mk.tex

```diff
@@ -189,7 +192,6 @@
 д1с2
 д1т2
 1ду
-1ѓ
```

Comment:
I guess this is the pattern you had to remove in order to make the patterns loadable in 8-bit encoding?
If so, we could add those patterns into an additional file, we already do that for a bunch of other languages where we duplicate apostrophe etc.

Reply:
Exactly. These 6 patterns (containing `ѓ` and `ќ`) were causing problems once I load:
```
\input conv-utf8-t2a.tex
\input hyph-mk.tex
```
before I removed them from `hyph-mk.tex`. It is not working because chars `ѓ` and `ќ` are not treated in any way in t2a.

I agree with you these 6 patterns can go in a separate file as soon as loading of  `conv-utf8-t2a.tex` and `hyph-mk.tex` do not fail.